pydis_site/apps/home/models/repo_data.py

```diff
@@ -0,0 +1,33 @@
+from django.db import models
+from django.utils import timezone
+
+
+class RepositoryMetadata(models.Model):
```

Comment:
you updated the model name but the module name should probably be updated to `repository_metdata.py` too then